meteor-app/imports/ui/assessment/assessment-add.js

```diff
@@ -203,6 +209,11 @@ class AssessmentAdd extends Component {
         services: minorServices,
         package: "Minor Service Package"
       },
+      costData: {
+        serviceCost: 6000,
```

Comment:
Prices should be in the /imports/api/constants.js file

Ideally they should be in the database, but I will allow just moving it to the constants file for now